### PR TITLE
Implement math opcodes

### PR DESCRIPTION
**Pull request by ShFil119**
While searching opcodes connected with text, I found math opcodes which can be easily implemented.

### CODE REVIEW

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 1]
```diff
@@ -8500,9 +8480,7 @@ void opcode_02f5(const ScriptArguments& args, const ScriptObject object, const S
 	@arg xOffset X offset
 */
 void opcode_02f6(const ScriptArguments& args, const ScriptFloat angle, ScriptFloat& xOffset) {
-	RW_UNIMPLEMENTED_OPCODE(0x02f6);
-	RW_UNUSED(angle);
-	RW_UNUSED(xOffset);
+	xOffset = glm::sin(angle);
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: The opcode argument is in degrees, but glm::sin takes radians. You can use glm::radians to convert the argument to rads.

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 2]
```diff
@@ -8514,9 +8492,7 @@ void opcode_02f6(const ScriptArguments& args, const ScriptFloat angle, ScriptFlo
 	@arg yOffset Y offset
 */
 void opcode_02f7(const ScriptArguments& args, const ScriptFloat angle, ScriptFloat& yOffset) {
-	RW_UNIMPLEMENTED_OPCODE(0x02f7);
-	RW_UNUSED(angle);
-	RW_UNUSED(yOffset);
+	yOffset = glm::cos(angle);
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: Same issue as 02f6

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 3]
```diff
@@ -12697,16 +12673,14 @@ bool opcode_0424(const ScriptArguments& args) {
...
 
 	opcode 0425
 	@arg arg1 
 	@arg arg2 
 */
 void opcode_0425(const ScriptArguments& args, const ScriptFloat arg1, ScriptFloat& arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x0425);
-	RW_UNUSED(arg1);
-	RW_UNUSED(arg2);
+	arg2 = arg1 * 3.2808398;
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: This is actually too accurate, the behaviour is documented on https://www.gtamodding.com/wiki/0425

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 4]
```diff
@@ -12793,16 +12767,14 @@ void opcode_042c(const ScriptArguments& args, const ScriptInt arg1) {
...
 
 	opcode 042d
 	@arg arg1 
 	@arg arg2 
 */
 void opcode_042d(const ScriptArguments& args, const ScriptInt arg1, ScriptInt& arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x042d);
-	RW_UNUSED(arg1);
-	RW_UNUSED(arg2);
+	arg2 = static_cast<ScriptInt>(std::round(static_cast<ScriptFloat>(arg1) * 3.2808398));
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: Same as 0425, the actual behaviour is slightly less accurate: https://www.gtamodding.com/wiki/042D

File: rwengine/src/script/modules/GTA3ModuleImpl.inl (hunk 3 above)
Comment (ShFil119) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: Multiplication with 3 will be enough?

File: rwengine/src/script/modules/GTA3ModuleImpl.inl (hunk 4 above)
Comment (ShFil119) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: What do you think about floor(arg1 * 3.3333334)?

Comment (danhedron) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: That seems close to the expected behaviour

File: rwengine/src/script/modules/GTA3ModuleImpl.inl (hunk 3 above)
Comment (danhedron) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: More like 3.333333, or division by 0.3

**Pull request merged**